Source/DafnyRuntime/DafnyRuntimeRust/src/tests/mod.rs

```diff
@@ -576,10 +582,10 @@ mod tests {
         assert_eq!(read!(c).constant_plus_x(), int!(82));
         modify_field!(read!(c).t, 54);
         assert_eq!(read_field!(read!(c).t), 54);
-        let x_copy = read_field!(read!(c).x);
-        assert_eq!(Rc::strong_count(&x_copy.data), 2);
-        deallocate(c);
-        assert_eq!(Rc::strong_count(&x_copy.data), 1);
+        // let x_copy = read_field!(read!(c).x);
```

Comment:
If this works only for regular ints but not small ints, please add a `#cfg(not(feature...` so that these lines are still taken into account.

Reply:
I had made `data` private, so I have no added a public `strong_count` method.